docs/contributing/code-reviewing.md

```diff
@@ -1,231 +1,364 @@
 
-- _Coding style._ See the Zulip [code-style] documentation for
-  details. Our goal is to have as much of this as possible verified
-  via the linters and tests, but there's always going to be unusual
-  forms of Python/JavaScript style that our tools don't check for.
+### Fast replies are key
 
-- _Clear commit messages._ See the [Zulip version
-  control][commit-messages] documentation for details on what we look
-  for.
+For the author of a PR, getting feedback quickly is really important
+for making progress quickly and staying productive. That means that
+if you get @-mentioned on a PR with a request for you to review it,
+it helps the author a lot if you reply promptly.
 
-### Zulip server
+A reply doesn't even have to be a full review; if a PR is big or if
+you're pressed for time, then just getting some kind of reply in
+quickly -- initial thoughts, feedback on the general direction, or
+just saying you're busy and when you'll have time to look harder -- is
+still really valuable for the author and for anyone else who might
+review the PR.
 
-Some points specific to the Zulip server codebase:
+People in the Zulip project live and work in many time zones, and code
+reviewers also need focused chunks of time to write code and do other
+things, so an immediate reply isn't always possible. But a good
+benchmark is to try to always reply **within one workday**, at least
+with a short initial reply, if you're working regularly on Zulip. And
+sooner is better.
 
-- _Testing -- Backend._ We are trying to maintain ~100% test coverage
-  on the backend, so backend changes should have negative tests for
-  the various error conditions.
+If a pull request just needs a little fixing to make it mergeable,
+feel free to do that in a new commit, then push your branch to GitHub
+and mention the branch in a comment on the pull request. That'll save
+the maintainer time and get the PR merged quicker.
 
-- _Testing -- Frontend._ If the feature involves frontend changes,
-  there should be frontend tests. See the [test
-  writing][test-writing] documentation for more details.
+### Responding to review feedback
 
-- _mypy annotations._ New functions should be annotated using [mypy]
-  and existing annotations should be updated. Use of `Any`, `ignore`,
-  and unparameterized containers should be limited to cases where a
-  more precise type cannot be specified.
+Once you've received a review and resolved any feedback, it's critical
+to update the GitHub thread to reflect that. Best practices are to:
 
-## Tooling
+- Make sure that CI passes and the PR is rebased onto recent `main`.
+- Post comments on each feedback thread explaining at least how you
+  resolved the feedback, as well as any other useful information
+  (problems encountered, reasoning for why you picked one of several
+  options, a test you added to make sure the bug won't recur, etc.).
+- Mark any resolved threads as "resolved" in the GitHub UI, if
+  appropriate.
```

Comment:
I think there was a proposal that contributors not do this, and only a second person verifying a change is correct should use this feature.

Reply:
Previous discussion on this was at https://chat.zulip.org/#narrow/stream/92-learning/topic/GitHub.20comment.20resolution

Reply:
Yeah, I recall that. Deleted.